packages/gatsby/src/pages/index.js

```diff
@@ -191,27 +196,27 @@ const IndexPage = ({data, searchState, onSearchStateChange}) => {
           <Section>
             <SectionContent>
               <SellingPoints>
-                <SellingPoint imgUrl={agendaIcon}>
+                <SellingPoint imgUrl={agendaIcon} url="/features/workspaces">
                   <h3>Workspaces</h3>
                   Split your project into sub-components kept within a single repository.
                 </SellingPoint>
-                <SellingPoint imgUrl={laptopIcon}>
+                <SellingPoint imgUrl={laptopIcon} url="/configuration/yarnrc#yarnUpgradePath">
                   <h3>Stability</h3>
                   Yarn guarantees that an install that works now will continue to work the same way in the future.
                 </SellingPoint>
-                <SellingPoint imgUrl={noteIcon}>
+                <SellingPoint imgUrl={noteIcon} url="/advanced/contributing">
```

Comment:
I'd point to to the `gatsby` folder of this repository (and we can add a README to explain what is it, plus pointing to the contributing guide).